wolfcrypt/src/asn.c

```diff
@@ -13121,6 +13122,36 @@ int wc_EncodeName(EncodedName* name, const char* nameStr, char nameType,
     return idx;
 }
 
+/* canonical encoding one attribute of the name (issuer/subject)
+ * call we_EncodeName_ex with CTC_UTF8 for email type
+ *
+ * name     structure to hold result of encoding
+ * nameStr  value to be encoded
+ * nameType type of encoding i.e CTC_UTF8
+ * type     id of attribute i.e ASN_COMMON_NAME
+ *
+ * returns length on success
+ */
+int wc_EncodeName_cano(EncodedName* name, const char* nameStr, char nameType,
```

Comment:
To match other ASN function naming conventions, I recommend we call this function "wc_EncodeNameCanonical".

Reply:
Agree to change the function name to "wc_EncodeNameCanonical".